src/Microsoft.TemplateEngine.Utils/FileSystemInfoExtensions.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
```

Comment:
nit: You are removing the UTF-8 BOM from the file. Is this an intentional change? Same for CreationPathModel.cs. Thanks!

![image](https://user-images.githubusercontent.com/12206368/94546121-c1144300-024d-11eb-9a09-cef43e86eca8.png)

Reply:
That's unintentional, I will fix this, thank you!